doc/source/conf.py

```diff
@@ -12,6 +12,7 @@
 #
 import os
 import sys
+from sphinx_pyproject import SphinxConfig
```

Comment:
We get an import error in the docs build test. Do we need to add sphinx_pyproject to the requirements in pyproject.toml, or where does this have to be installed?

Reply:
The pipeline to check the docs never installed the docs dependencies. I have now updated this.

I also removed the explicit install of the master branch of pandapower. This is a dependencie and should be installed from the projects config to avoid issues from changes in pandapower that have not been acounted for in pandapipes